Source/WebCore/html/HTMLMediaElement.h

```diff
@@ -759,6 +759,8 @@ class HTMLMediaElement
     bool videoFullscreenStandby() const { return m_videoFullscreenStandby; }
     void setVideoFullscreenStandbyInternal(bool videoFullscreenStandby) { m_videoFullscreenStandby = videoFullscreenStandby; }
 
+    void beginIgnoringFullscreenPermissionPolicy() { m_ignoreFullscreenPermissionsPolicy = true; }
```

Comment:
Nit: seems fine as-is, but maybe `ignoreFullscreenPermissionPolicyOnNextCallToEnterFullscreen` better captures that this value is reset when `enterFullscreen` is called?